src/lib/services/invoice-accounts-service.js

```diff
@@ -153,12 +152,13 @@ const persist = async (regionId, startDate, invoiceAccount) => {
     await createCompanyAddressAndContact(invoiceAccount.company.id, formattedStartDate, invoiceAccount, newModels);
 
     await createInvoiceAccount(regionId, invoiceAccount.company.id, formattedStartDate, invoiceAccount, newModels);
-
     const invoiceAccountAddress = await invoiceAccountAddressesService.createInvoiceAccountAddress(invoiceAccount, invoiceAccount.invoiceAccountAddresses[0], formattedStartDate);
+
     return invoiceAccount.fromHash({
       invoiceAccountAddresses: [invoiceAccountAddress]
     });
   } catch (err) {
+    console.log(err);
```

Comment:
Did you mean to leave this in?

Reply:
Yes - But I'll change it to logger() as per our chat :)